earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,65 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
+    packageName = openGeeCommonRpm.packageName
+    release = openGeeCommonRpm.release
+    version = openGeeVersion
+    user = openGeeCommonRpm.user
+    permissionGroup = openGeeCommonRpm.permissionGroup
+    packageGroup = 'misc'
+    summary = openGeeCommonRpm.summary
+    // The package build failed when packageDescription was copied here from openGeeCommonRpm, so it is omitted for now
+    license = openGeeCommonRpm.license
+    priority = 'optional'
+
+    setArch('amd64')	// if this is not specified, the architecture is x86_64, and dpkg won't install without the --force-architecture flag
```

Comment:
Yes, the correct value is `amd64`.

There's no `x86_64` architecture string in Debian packages.  According to the Debian Policy Manual (https://www.debian.org/doc/debian-policy/#architecture-specification-strings), you can get the list of recognized architecture strings with `dpkg-architecture -L`.